samples/Samples.Server/CustomErrorInfoProvider.cs

```diff
@@ -26,7 +26,7 @@ private string GetAuthorizationErrorMessage(AccessDeniedError error)
         var errorMessage = new StringBuilder();
         errorMessage.Append(error.Message);
 
-        foreach (var failedRequirement in error.PolicyAuthorizationResult.Failure.FailedRequirements)
+        foreach (var failedRequirement in error.PolicyAuthorizationResult!.Failure!.FailedRequirements)
```

Comment:
Why `!` operator?

Reply:
`error.PolicyAuthorizationResult` may be null if a role check failed rather than a policy check.  `Failure` is defined as nullable by .NET because `AuthorizationResult` may be a successful result -- but it will never be so here.

This code is old unchanged code.  I'll wrap the `switch` in an `if` to make a better sample.  `Failure!` is still proper.

Reply:
ok